draft-ietf-httpbis-priority.md

```diff
@@ -205,6 +205,17 @@ Where both endpoints disable HTTP/2 priorities, the client is expected to send
 this scheme's priority signal. Handling of omitted signals is described in
 {{parameters}}.
 
+# Applicability of the Extensible Priority Scheme
+
+The priority scheme defined by this document considers only the prioritization
+of HTTP response data. Clients express priority values to a server, which are
+input into scheduling decisions, see {{client-scheduling}} and
+{{server-scheduling}}.
+
+Where HTTP extensions change stream behavior or define new data carriage
+mechanisms, they MAY also define how this priority scheme can be applied.
```

Comment:
This paragraph LGTM.

Reply:
FWIW, I would prefer s/MAY/can/ - there is no normative effect of a choice.

Reply:
missed this while tagging the release sorry. I'll create a followup ticket to roll into next round of editorial changes.

Reply:
see https://github.com/httpwg/http-extensions/issues/1571